libgc/gcj_mlc.c

```diff
@@ -50,6 +50,8 @@ int GC_gcj_debug_kind;	/* The kind of objects that is always marked 	*/
 ptr_t * GC_gcjobjfreelist;
 ptr_t * GC_gcjdebugobjfreelist;
 
+void GC_start_debugging();
```

Comment:
What is this needed for?

Reply:
This function is used in this file further down the line. It's never declared. Technically it's legal to call undeclared functions in C (the compiler just assumes they take no arguments and return int) but the latest Mac SDK doesn't like this and errors out.

Reply:
Ah, this is old boehm. We certain don't need this to build at all. Perhaps it is built, but it should never be used by anything in Unity.

Reply:
Correct. I asked you about it a few months ago, you said you just never got around to disable it :).